src/api-reference/common/list-item/v4.list-item-bulk.markdown

```diff
@@ -0,0 +1,494 @@
+
+* [Bulk List Item Create Request](#schema-list-item-create-request)
+* [Bulk List Item Create Request Part](#schema-list-item-create-request-part)
+
+### Response
+
+#### Status Codes
+
+* [201 Created](https://tools.ietf.org/html/rfc7231#section-6.3.2)
+* [206 Partial Content](https://www.rfc-editor.org/rfc/rfc7233#section-4.1)
+* [400 Bad Request](https://tools.ietf.org/html/rfc7231#section-6.5.1)
+* [401 Unauthorized](https://tools.ietf.org/html/rfc7235#section-3.1)
+* [403 Forbidden](https://tools.ietf.org/html/rfc7231#section-6.5.3)
+* [404 Not Found](https://tools.ietf.org/html/rfc7231#section-6.5.4)
+* [415 Unsupported Media Type](https://tools.ietf.org/html/rfc7231#section-6.5.13)
+* [500 Internal Server Error](https://tools.ietf.org/html/rfc7231#section-6.6.1)
+
+#### Headers
+
+* `concur-correlationid` is a Concur specific custom header used for technical support in the form of a [RFC 4122 A Universally Unique IDentifier (UUID) URN Namespace](https://tools.ietf.org/html/rfc4122)
+* [RFC 7230 Content-Length](https://tools.ietf.org/html/rfc7230#section-3.3.2)
+* [RFC 7231 Content-Type](https://tools.ietf.org/html/rfc7231#section-3.1.1.5)
+* [RFC 7231 Date](https://tools.ietf.org/html/rfc7231#section-7.1.1.2)
+* [RFC 7231 Location](https://tools.ietf.org/html/rfc7231#section-7.1.2)
+* [RFC 7234 Cache-Control](https://tools.ietf.org/html/rfc7234#section-5.2)
+
+#### Payload
+
+* [Bulk List Item Response](#schema-bulk-list-item-response)
+* [Bulk List Error](#schema-bulk-list-error)
+* [Bulk List Item Request](#schema-bulk-list-item-request)
+* [Error Message](#schema-error-message)
+
+### Example
+
+#### Request
+
+```http
+POST https://us.api.concursolutions.com/list/v4/lists/530ae443-3b22-a44c-8211-c66d96c58d9e/bulk
+Accept: application/json
+Accept-Language: en
+Content-Type: application/json
+Authorization: Bearer {token}
+```
+Example 1:
+```json
+{
+  "requests": [
+    {
+      "shortCode": "ITEM",
+      "value": "ITEM"
+    }
+  ]
+}
+```
+
+Example 2:
+```json
+{
+  "requests": [
+    {
+      "shortCode": "CHILD_ITEM",
+      "value": "CHILD_ITEM",
+      "parentCode": "ITEM"
+    }
+  ]
+}
+```
+
+Example 3:
+```json
+{
+  "requests": [
+    {
+      "shortCode": "CHILD_ITEM_ONE",
+      "value": "CHILD_ITEM_ONE",
+      "parentCode": "ITEM"
+    },
+    {
+      "shortCode": "CHILD_ITEM_B",
+      "value": "CHILD_ITEM_B",
+      "parentCode": "ITEM"
+    },
+    {
+      "shortCode": "ITEM_THREE",
+      "value": "ITEM_THREE"
+    }
+  ]
+}
+```
+
+#### Response
+
+```shell
+HTTP/1.1 201
+concur-correlationid: 610425c5-4c2e-4bdb-ab5d-02775e23c7c9 
+content-length: 71 
+content-type: application/json;charset=UTF-8
+date: Fri,26 Aug 2022 16:13:17 GMT 
+location: https://us.api.concursolutions.com/list/v4/lists/530ae443-3b22-a44c-8211-c66d96c58d9e/bulk
+cache-control: no-cache, private
+```
+Example 1:
+```json
+{
+  "status": "SUCCESS",
+  "recordsSucceeded": 1,
+  "recordsFailed": 0,
+  "errors": []
+}
+```
+
+Example 2:
```

Comment:
You will probably need to also include the HTTP response header including the status code for examples 2 and 3 since the HTTP status code will be different than 201.